Library/Homebrew/extend/os/linux/extend/ENV/super.rb

```diff
@@ -40,8 +40,11 @@ def homebrew_extra_isystem_paths
     paths = []
     # Add paths for GCC headers when building against glibc@2.13 because we have to use -nostdinc.
     if deps.any? { |d| d.name == "glibc@2.13" }
-      gcc_include_dir = Utils.safe_popen_read(cc, "--print-file-name=include").chomp
-      gcc_include_fixed_dir = Utils.safe_popen_read(cc, "--print-file-name=include-fixed").chomp
+      env_cc = cc
```

Comment:
We need to store `cc` and use a type guard, as sorbet doesn't assume successive invocations have the same return type.